demos/default/netlify.toml

```diff
@@ -1,7 +1,7 @@
 [build]
-command = "next build"
-publish = ".next"
-ignore = "if [ $CACHED_COMMIT_REF == $COMMIT_REF ]; then (exit 1); else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF ../..; fi;"
```

Comment:
We actually want to get a diff for the whole repo and not `git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF demos/canary` which would the regular diff for monorepos